FraudProofs.md

```diff
@@ -0,0 +1,154 @@
+If `num_oracles = 1`, then the `aggregate_oracle_attestation` is directly equal to the attestation released by the one oracle. 
+As such, if one has access directly to an oracle's attestation, then this proof should use `num_oracles = 1`.
+
+The `oracle_announcements` and `oracle_outcomes` are used to compute a signature point `S` corresponding to an anticipation of
+these oracles attesting to these outcomes.
+This signature point is used to validate that `aggregate_oracle_attestation*G = S` which proves that there must exist an attestation
+from each of the oracles in `oracle_announcements` of their corresponding outcome in `oracle_outcomes`.
+
+Finally, this proof references a specific oracle by `oracle_index` which is the index in `oracle_announcements` and `oracle_outcomes`
+corresponding to the referenced oracle's announcement and outcome to which they attested. 
+
+## Claim of False Oracle Attestation
+
+The most straightforward kind of oracle fraud occurs when an oracle attests to a false event outcome,
+meaning that the attested-to outcome is not what truly occurred in the real world.
+
+Unfortunately, there is no general and automatic way to validate a claim of this kind of fraud as there
+is often times some amount of subjectivity involved or at the very least validation must compare what
+the oracle said to the real world, a process which usually requires manual human labor.
+
+As such, this specification only provides a standard format for making such claims which is generally
+intended for use with other external infrastructure.
+That said, it is possible for DLC client software to accept the manual importing of these claims after
+which user's can be taken through manual validation step-by-step.
+Such a process can serve as a manual means of black-listing oracle keys.
+
+#### Version 0 `false_oracle_attestation_claim`
+
+1. type: ??? (`false_oracle_attestation_claim_v0`)
+2. data:
+   * [`oracle_attestation_proof`:`oracle_attestation_proof`]
+   * [`string`:`fraud_claim`]
+
+This message consists of an `oracle_attestation_proof` showing that a specific oracle attested to
+a specific message and a `fraud_claim` explaining why this attestation is fraudulent.
+
+## Proof of Invalid Oracle Attestation
+
+The next simplest form of fraud is if an oracle attests using some outcome outside of its announced
+and committed to outcome set from its `oracle_announcement`.
+
+Unlike false attestations, this kind of fraud can be automatically validated by client software.
+
+#### Version 0 `invalid_oracle_attestation_proof`
+
+1. type: ??? (`invalid_oracle_attestation_proof_v0`)
+2. data:
+   * [`oracle_attestation_proof`: `oracle_attestation_proof`]
+
+This type simply consists of an `oracle_attestation_proof` proving that a specific oracle attested to
+a specific `oracle_outcome` where this outcome is not allowed by that oracle's `oracle_announcement`.
```

Comment:
Again, can you define what's an `oracle_outcome` and from where it's discovered :) ?